elkodon/tests/publisher_tests.rs

```diff
@@ -88,15 +83,13 @@ mod publisher {
 
         let sut = service.publisher().max_loaned_samples(2).create().unwrap();
 
-        let _sample1 = sut.loan().unwrap();
-        let mut sample2 = sut.loan().unwrap();
-        sample2.payload_mut().write(2);
-        let sample2 = unsafe { sample2.assume_init() };
+        let _sample1 = sut.loan_uninit().unwrap();
```

Comment:
Could you please either add a test that also uses the `loan()` API or modify an existing test so that this API is used at least once.

Reply:
Hahaha, I had it and then removed it because there was a documentation example. I can re-add it though.